screens/ai_recommendations.py

```diff
@@ -0,0 +1,99 @@
+import streamlit as st
+import plotly.graph_objs as go
+from utilz.llama_response import get_ai_recommendations_from_api
```

Comment:
this is actually Gemini, not llama, it was my bad - I never corrected it. if it wasn't a demo the function name should have been left generic for good measure.